src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs

```diff
@@ -0,0 +1,283 @@
+
+				ColorPickerSliderAutomationPeer peer = FrameworkElementAutomationPeer.FromElement(this) as ColorPickerSliderAutomationPeer;
+				peer.RaisePropertyChangedEvent(oldColor, newColor, (int)(Math.Round(args.OldValue)), (int)(Math.Round(args.NewValue)));
+			}
+		}
+
+		private ColorPicker GetParentColorPicker()
+		{
+			ColorPicker parentColorPicker = null;
+			DependencyObject currentObject = this;
+
+			while (currentObject != null && !(parentColorPicker is ColorPicker))
+			{
+				currentObject = VisualTreeHelper.GetParent(currentObject);
+			}
+
+			return parentColorPicker;
+		}
+
+		private string GetToolTipString()
+		{
+			uint sliderValue = (uint)(Math.Round(this.Value));
+
+			if (this.ColorChannel == ColorPickerHsvChannel.Alpha)
+			{
+				return string.Format(
+					ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringAlphaSlider),
+					sliderValue);
+			}
+			else
+			{
+				ColorPicker parentColorPicker = GetParentColorPicker();
+				if (parentColorPicker != null && DownlevelHelper.ToDisplayNameExists())
+				{
+					Hsv currentHsv = parentColorPicker.GetCurrentHsv();
+					string localizedString;
+
+					switch (this.ColorChannel)
+					{
+						case ColorPickerHsvChannel.Hue:
+							currentHsv.H = this.Value;
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringHueSliderWithColorName);
+							break;
+
+						case ColorPickerHsvChannel.Saturation:
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringSaturationSliderWithColorName);
+							currentHsv.S = this.Value / 100;
+							break;
+
+						case ColorPickerHsvChannel.Value:
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringValueSliderWithColorName);
+							currentHsv.V = this.Value / 100;
+							break;
+						default:
+							throw new Exception(); // Uno Doc: 'throw winrt::hresult_error(E_FAIL);'
+					}
+
+					return string.Format(
+						localizedString,
+						sliderValue,
+						ColorHelper.ToDisplayName(ColorConversion.ColorFromRgba(ColorConversion.HsvToRgb(currentHsv))));
+				}
+				else
+				{
+					string localizedString;
+					switch (this.ColorChannel)
+					{
+						case ColorPickerHsvChannel.Hue:
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringHueSliderWithoutColorName);
+							break;
+						case ColorPickerHsvChannel.Saturation:
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringSaturationSliderWithoutColorName);
+							break;
+						case ColorPickerHsvChannel.Value:
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringValueSliderWithoutColorName);
+							break;
+						default:
+							throw new Exception(); // Uno Doc: 'throw winrt::hresult_error(E_FAIL);'
+					}
+
+					return string.Format(
```

Comment:
Codacy found an issue: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)